src/Core/Package.vala

```diff
@@ -677,7 +681,11 @@ public class AppCenterCore.Package : Object {
             }
 
             try {
+#if HAS_APPSTREAM_1_0
+                description = AppStream.markup_convert (description, AppStream.MarkupKind.XML);
```

Comment:
I'm assuming we only convert XML 🤔

Reply:
The 2nd parameter is the format to convert _to_.

`markup_convert_simple` in AppStream < 1.0 used to convert to the `TEXT` markup kind, so this would be a difference in behaviour with the Appstream 1.0 code and should probably be changed to `TEXT`.

That said, it feels like we'd want to keep the description as markup so we can display it more richly in the interface, but I think checking and testing that is outside the scope of supporting AppStream 1.0

```suggestion
                description = AppStream.markup_convert (description, AppStream.MarkupKind.TEXT);
```